README.md

```diff
@@ -2,13 +2,51 @@
 
 ## Introduction
 
-Goordinator listens for GitHub webhook events, runs their JSON payloads through
-a JQ filter query and triggers actions if the filter matches.
+Goordinator is an event-processor for GitHub events.
+It provides 2 functionalities.
+
+### Configurable Event-Trigger Loop
+
+Goordinator listens for GitHub webhook events, runs their JSON payloads
+through a JQ filter query and triggers actions if the filter matches.
+The supported actions are:
+- posting a http-request
+- updating a GitHub branch with it's base branch.
 
-The only supported action is currently posting HTTP-Requests.
 All actions are executed in parallel and retried if they fail until a retry
 timeout expired (default: 2h).
 
+### Serialized Synchronization of GitHub Pull Request with their Base Branch
+
+Autoupdater keeps pull requests (PR) updated with their base branch. \
+Pull requests are added to a per base branch queue and the first pull request in
+the queue is kept up to date with its base branch. \
+Updates for the PR branch are suspended when:
+
+- the base-branch can not be merged into the PR branch because of a
+  merge-conflict,
+- its GitHub check status becomes negative or
+- it became stale and its status have not changed for a longer time period.
```

Comment:
```suggestion
- it became stale and its status has not changed for a long time period.
```

Reply:
I would prefer to keep it unspecific in the README (otherwise I have to keep those details uptodate). The idea was to have in the README only a short summarized subscription of the features.
Later I can either link the godoc or create a more end-user friendly detailed documentation in the wiki.

The long time period is 3h (https://github.com/simplesurance/goordinator/blob/810a251ddcbbdf9c8e72c743fd88d29dda514c2a/internal/autoupdate/queue.go#L21) :-)